trunk/ghost/ghost.c

```diff
@@ -913,7 +929,7 @@ static void Ghost_Command_f (void)
 
     DZip_Cleanup(&ghost_dz_ctx);
 
-    if (ok && (sv.active || cls.demoplayback)) {
+    if (ok && cls.state == ca_connected) {
         Ghost_UpdateMarathon();
```

Comment:
We want to update this stuff even when connected as a non-host client.  This would make a difference in co-op when changing ghost on the intermission screen for example --- without this change a non-host would not see the times change after loading the new ghost.